styles/styles.css

```diff
@@ -0,0 +1,84 @@
+@import "section-1.css";
+@import "section-2.css";
+@import "section-3.css";
+@import "section-4.css";
+@import "section-5.css";
+@import "footer.css";
+
+body {
+  padding: 0;
+  margin: auto;
+}
+
+nav {
+  width: 1349px;
+  height: 42px;
+  position: fixed;
+  top: 0;
+  display: flex;
+  align-items: center;
+  justify-items: center;
+  background-color: hsl(0, 0%, 100%);
+  border-bottom: 1px solid #e2e2e2;
+}
```

Comment:
2 details with your navbar:
- [ ] **[OPTIONAL]** Your navbar looks incomplete (it may be because my screen is larger) but to ensure that it is always the full screen, you can use `width: 100%;`
- [ ] **[OPTIONAL]** Also, your `justify-items: center;` is not doing anything, you positioned everything with margins. If you want to center with flex, use `justify-content: center;`
![image](https://user-images.githubusercontent.com/53256139/88502659-c4145e00-cf94-11ea-85ac-710a381f1d69.png)